src/ansys/tools/repo_sync/repo_sync.py

```diff
@@ -32,6 +33,37 @@ def _ignore_patterns(path, names):
     return _ignore_patterns
 
 
+def delete_folder_contents(folder_path):
+    """Delete the content inside a folder without deleting the folder itself.
+
+    Parameters
+    ----------
+    folder_path : str
+        Path to the folder whose content is requested to be deleted.
+
+    """
+    # Check if the folder exists
+    if not os.path.exists(folder_path):
+        return
+
+    try:
+        # List all files and directories in the folder
+        for item in os.listdir(folder_path):
+            item_path = os.path.join(folder_path, item)
+
+            if os.path.isfile(item_path):
+                # If it's a file, delete it
+                os.remove(item_path)
+            elif os.path.isdir(item_path):
+                # If it's a directory, use recursive call to delete its contents
+                delete_folder_contents(item_path)
+                # After the contents are deleted, remove the empty directory
+                os.rmdir(item_path)
+
+    except Exception as e:  # pragma: no cover

```

Comment:
I would probably use specific exceptions such as FileNotFound, FileIOError, etc...

Reply:
Yep, I thought about that too... Also adding it to the list.